### PR TITLE
deals: Ask prices per GiB

### PR DESCRIPTION
**Pull request by magik6k**

### CODE REVIEW

File: api/api.go [hunk 1]
```diff
@@ -206,8 +206,8 @@ type DealInfo struct {
 	PieceRef []byte // cid bytes
 	Size     uint64
 
-	TotalPrice types.BigInt
-	Duration   uint64
+	PricePerEpoch types.BigInt
```
Comment (magik6k) [inline api/api.go]: This matches the spec more closely (and removes one BigDiv from actor code)

File: chain/actors/actor_storagemarket.go [hunk 2]
```diff
@@ -85,12 +85,16 @@ type StorageDealProposal struct {
 	//  Changing to duration makes sure that the price-per-block is defined, and the miner
```
Comment (Kubuxu) [inline chain/actors/actor_storagemarket.go]: Comment is obsolete.

File: cli/client.go [hunk 3]
```diff
@@ -314,13 +314,14 @@ var clientQueryAskCmd = &cli.Command{
 		if size == 0 {
 			return nil
 		}
-		fmt.Printf("Price per Block: %s\n", types.BigMul(ask.Ask.Price, types.NewInt(uint64(size))))
+		perEpoch := types.BigDiv(types.BigMul(ask.Ask.Price, types.NewInt(uint64(size))), types.NewInt(1<<30))
+		fmt.Printf("Price per Block: %s\n", perEpoch)
```
Comment (Kubuxu) [inline cli/client.go]: Use `types.Fil(perEpoch)` for nice print.

File: cli/client.go [hunk 4]
```diff
@@ -314,13 +314,14 @@ var clientQueryAskCmd = &cli.Command{
...
 
 		duration := cctx.Int64("duration")
 		if duration == 0 {
 			return nil
 		}
-		fmt.Printf("Total Price: %s\n", types.BigMul(types.BigMul(ask.Ask.Price, types.NewInt(uint64(size))), types.NewInt(uint64(duration))))
+		fmt.Printf("Total Price: %s\n", types.BigMul(perEpoch, types.NewInt(uint64(duration))))
```
Comment (Kubuxu) [inline cli/client.go]: Same here.

**Pull request merged**